Makefile

```diff
@@ -37,7 +37,7 @@ endif
 GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
 IMAGE := skopeo-dev$(if $(GIT_BRANCH),:$(GIT_BRANCH))
 # set env like gobuildtag?
-CONTAINER_CMD := ${CONTAINER_RUNTIME} run --rm -i -e TESTFLAGS="$(TESTFLAGS)" #$(CONTAINER_ENVS)
+CONTAINER_CMD := ${CONTAINER_RUNTIME} run --privileged --rm -i -e TESTFLAGS="$(TESTFLAGS)" #$(CONTAINER_ENVS)
```

Comment:
Do the existing root-run tests really need `--privileged` now?

Reply:
no, I can tune it later if this works, probably I'll need to enable some more syscalls in the seccomp profile that are used when running as rootless

Reply:
My first impression is that the `--privileged` should only be necessary in `CONTAINER_RUN_ROOTLESS`, not in `CONTAINER_CMD`.

Sure, if rootless tests need extra privileges, fine.